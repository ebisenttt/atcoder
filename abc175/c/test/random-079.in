-842653892556431 43553206003448 390054667231510
