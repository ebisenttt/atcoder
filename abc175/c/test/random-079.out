62544558093411
